src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/postsubmit/AutomaticEndAppealForNonPaymentEaHuTrigger.java

```diff
@@ -65,14 +101,14 @@ public PostSubmitCallbackResponse handle(
         ZonedDateTime scheduledDate = ZonedDateTime.of(dateProvider.nowWithTime(), ZoneId.systemDefault()).plusMinutes(schedule14DaysInMinutes);
 
         scheduler.schedule(
-            new TimedEvent(
-                "",
-                Event.END_APPEAL_AUTOMATICALLY,
-                scheduledDate,
-                "IA",
-                "Asylum",
-                callback.getCaseDetails().getId()
-            )
+                new TimedEvent(
```

Comment:
Please change the IDE settings or while doing git diff remove these changes.

Reply:
resolved.